sections/03_categorize.md

```diff
@@ -63,6 +63,29 @@ subsequent training. Similar strategies have been employed for EHR data where
 high-quality labeled examples are also difficult to obtain
 [@doi:10.1101/039800].
 
+Chest X-ray image analysis is another area where recent DL methods showed
```

Comment:
As noted elsewhere, we'll need to standardize over the various methods of deep learning / Deep Learning / DL. Suggest DL -> deep learning.